Source/Particles/ElementaryProcess/QEDInternals/SchwingerProcessWrapper.H

```diff
@@ -49,10 +50,10 @@ getSchwingerProductionNumber (const amrex::Real dV, const amrex::Real dt,
             Ex, Ey, Ez, Bx, By, Bz, dV, dt);
 
     if (expectedPairNumber <= PoissonToGaussianThreshold) {
-        return amrex::RandomPoisson(expectedPairNumber);
+        return amrex::RandomPoisson(expectedPairNumber, engine);
     } else {
         const auto numpairs =
-            amrex::RandomNormal(expectedPairNumber,sqrt(expectedPairNumber));
+            amrex::RandomNormal(expectedPairNumber,std::sqrt(expectedPairNumber));
```

Comment:
Does `amrex::RandomNormal` also need an `engine` object?

Reply:
Yes, it does need an `engine` object.  This is a new line of code from merging development into this branch, but I forgot to update it.  Thanks!